### PR TITLE
Feature typescript definition files

### PR DESCRIPTION
**Pull request by hdaljord**
Partially solves issue #598. This is just a start, I'd like to add more types to other react-components, but would like to have some feedback on these first.

### CODE REVIEW

**Review by ivarni — DISMISSED** [review]
I know nothing about TypeScript but it looks reasonable to me.

I guess this means everyone who makes changes to these packages must keep these files in mind if they add, remove or change properties. Is there any way we can help people be aware of that?

**Comment by hdaljord** [comment]
> I know nothing about TypeScript but it looks reasonable to me.
> 
> I guess this means everyone who makes changes to these packages must keep these files in mind if they add, remove or change properties. Is there any way we can help people be aware of that?

That's a good point, would it be sufficient to add a note to readme.md? I can look around a bit and see if there are any good automatic solutions as well.

File: packages/ffe-grid-react/src/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    topPadding?: boolean;
+}
+
+export interface InlineGridProps extends React.HTMLAttributes<HTMLElement> {
+    children: React.ReactNode;
+    className?: string;
+    element?: string;
+}
+
+type backgroundColors =
```
Comment (Saegrov) [inline packages/ffe-grid-react/src/index.d.ts]: Nitpicking but I think it is normal to use PascalCase also for types.

Comment (hdaljord) [reply packages/ffe-grid-react/src/index.d.ts]: I'll fix :)

**Comment by Saegrov** [comment]
LGTM!

**Comment by hdaljord** [comment]
I couldn't find any tools to help the sync issue, but I added a few notes to the readme-files and contribution-documentation.

**Comment by hdaljord** [comment]
Travis fails on linting of a file that hasn't been changed.

**Comment by ivarni** [comment]
@hdaljord I'm pretty sure I've fixed that yesterday. Does rebasing from master help?

https://github.com/SpareBank1/designsystem/commit/350fd75d4d94e5087b12b060d5a789ecb790bfb2

**Comment by hdaljord** [comment]
> @hdaljord I'm pretty sure I've fixed that yesterday. Does rebasing from master help?
> 
> [350fd75](https://github.com/SpareBank1/designsystem/commit/350fd75d4d94e5087b12b060d5a789ecb790bfb2)

Yes and no :/ The ffe-header-react linting error disappears, but it gives a few new ones in ffe-lists-react. These is on develop as well.

**Comment by ivarni** [comment]
Might just have to fix it then, the linting in this project kind of does its own thing whenever it feels like it

**Review by ivarni — APPROVED** [review]

**Pull request merged**